pkg/helper/tokens_test.go

```diff
@@ -0,0 +1,26 @@
+package helper
+
+import (
+	"fmt"
+	"testing"
+)
+
+//Maybe more thorough testing? Unsure how to test random numbers...
```

Comment:
You might want to test that you got back the expected length.  Not required for this but since you asked a question, I figured I'd provide an idea.